src/Private/Middleware.ps1

```diff
@@ -167,7 +167,7 @@ function Get-PodeLimitMiddleware {
             }
 
             # check the endpoint
-            if (!(Test-PodeEndpointLimit -EndpointName $WebEvent.Endpoint.Name)) {
+            if (!(Test-PodeEndpointByProtocolTypeLimit -EndpointName $WebEvent.Endpoint.Name)) {
```

Comment:
This function looks like it might have gotten cause in a `Test-PodeEndpointByProtocol` find+replace :)